huaweicloud/services/mrs/resource_huaweicloud_mapreduce_cluster.go

```diff
@@ -415,6 +421,75 @@ func externalDatasourceSchema() *schema.Resource {
 	}
 }
 
+func bootstrapScriptsSchema() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Name of a bootstrap action script.`,
+			},
+			"uri": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Path of a bootstrap action script. Set this parameter to an OBS bucket path or a local VM path.`,
+			},
+			"nodes": {
+				Type:        schema.TypeList,
```

Comment:
shall we use `typeSet`?

Reply:
no, the order in response is the same as in request